Zend/tests/type_declarations/scalar_none.phpt

```diff
@@ -8,10 +8,10 @@ $functions = [
     'float' => function (float $f) { return $f; },
     'string' => function (string $s) { return $s; },
     'bool' => function (bool $b) { return $b; },
-    'int nullable' => function (int $i = NULL) { return $i; },
-    'float nullable' => function (float $f = NULL) { return $f; },
-    'string nullable' => function (string $s = NULL) { return $s; },
-    'bool nullable' => function (bool $b = NULL) { return $b; }
+    'int nullable' => function (?int $i = NULL) { return $i; },
+    'float nullable' => function (?float $f = NULL) { return $f; },
+    'string nullable' => function (?string $s = NULL) { return $s; },
+    'bool nullable' => function (?bool $b = NULL) { return $b; }
```

Comment:
Removing the default values would change the test expectation, since the closures are called with 0 arguments, so I'm not sure if it's really a good idea. What do you think?

Reply:
Ah right, it's testing 0 param, yes leaving the default value is fine then :)